app/src/main/java/org/oppia/android/app/walkthrough/topiclist/WalkthroughTopicListFragment.kt

```diff
@@ -31,4 +31,9 @@ class WalkthroughTopicListFragment : InjectableFragment(), TopicSummaryClickList
   override fun onTopicSummaryClicked(topicSummary: TopicSummary) {
     walkthroughTopicListFragmentPresenter.changePage(topicSummary)
   }
+
+  override fun onStop() {
+    super.onStop()
+    walkthroughTopicListFragmentPresenter.hideHeaderAndShowProgressBar()
```

Comment:
Do we need this or do we have some other way to do it?

Reply:
@anandwana001 I changed it a bit, PTAL

Reply:
I am not sure about this, it seems like some lifecycle issue.